ang/volunteer/Project.html

```diff
@@ -0,0 +1,120 @@
+<div class="crm-container">
+  <div crm-ui-debug="project"></div>
+  <div crm-ui-debug="relationships"></div>
+  <div crm-ui-debug="profiles"></div>
+  <div crm-ui-debug="locBlock"></div>
+  <div id="backboneIncludes"></div>
+
+  <form class="crm-form-block" name="projectForm" crm-ui-id-scope>
+
+    <div class="help"></div>
+
+    <fieldset>
+      <legend>{{ ts("Project Settings") }}</legend>
+      <div crm-ui-field="{name: 'projectForm.title', title: ts('Project Title')}">
+        <input crm-ui-id="projectForm.title" ng-model="project.title" ng-required="true" />
+      </div>
+
+      <div crm-ui-field="{name: 'projectForm.description', title: ts('Project Description')}">
+        <textarea crm-ui-id="projectForm.description" ng-model="project.description"></textarea>
+      </div>
+
+      <div crm-ui-field='{name: "projectForm.campaign", title: ts("Campaign")}'>
+        <input crm-ui-id="projectForm.campaign" crm-entityref="{entity: 'Campaign', select: {allowClear:true}}" ng-model="project.campaign" />
+      </div>
+
+      <div crm-ui-field='{name: "projectForm.locBlock", title: ts("Location Block")}'>
+        <select ng-change="locBlockChanged()" crm-ui-select="{placeholder: '', allowClear:true, width: '90%'}" ng-options="key as value for (key , value) in locationBlocks" ng-model="project.loc_block_id">
+          <option />
+        </select>
+        <div id="crm-vol-location-block" class="crm-vol-extra-top" crm-ui-accordion="{title: ts('Edit Location'), collapsed: project.loc_block_id !==0}" ng-hide="!project.loc_block_id">
+          <div class="status" ng-show="locBlock.count_loc_used > 0">{{ts("This location is used by other events/projects. Modifying location information will change values for all events/projects")}}</div>
```

Comment:
I don't see how ng-show would ever evaluate to true. Maybe you meant to be doing an api.LocBlock.getrefcount somewhere?